### PR TITLE
Improve dev documentation and tweaks to handle Windows host machines

### PR DESCRIPTION
**Pull request by rob-metalinkage**
A series of minor fixes and instructions to provisioning scripts to 
a) work with a Windows Host machine
b) allow reboot of guest vm

with general improvements to the documentation to identify roles of components,

### CODE REVIEW

File: scripts/provision/roles/web/tasks/mapstory.yml [hunk 1]
```diff
@@ -67,13 +68,24 @@
...
+  tags: [settings]
+ 
+- name: write project local settings - sync filesystem safe
+  template: src=roles/web/files/local_settings.py.j2 dest=/tmp/local_settings.py
+  sudo_user: '{{ application_user }}'
+  tags: [settings]
 
-- name: write project local settings
-  template: src=roles/web/files/local_settings.py.j2 dest={{ project_path }}/{{ project_name }}/settings/local_settings.py
+- name: mv project local settings across synced file system if necessary
```
Comment (davisc) [inline scripts/provision/roles/web/tasks/mapstory.yml]: @rob-metalinkage how is this used?

Comment (rob-metalinkage) [inline scripts/provision/roles/web/tasks/mapstory.yml]: during provisioning it avoids a known limitation of django manage to write to a synced filesystem. Just puts output somewhere tmp then moves it to the desired target.

**Comment by davisc** [comment]
@rob-metalinkage if you can update this branch to resolve the conflicts, we can get this merged in.

**Comment by rob-metalinkage** [comment]
I think I need a little more understanding of the process here - how do i see the conflicts and what causes them? 
 - when i generated the the pull was there an incompatible change in the upstream i didnt take into account? I thought the pull reported that it could be merged - or was it changes made to my forked master branch since then - and i need to make a branch per pull request so there is a stable target for the pull request.?

**Comment by rob-metalinkage** [comment]
anyway - I still havent quite reached the goal of a dev environment that has documented and repeatable processes for:
1) provisioning
2) surviving restart of the vm
3) propagating updates from imported components
4) debugging the data upload process.

For example, after merging latest and reprovisioning celery has failed to start, an uploader fails without any diagnostic information.

Is it worthwhile doing incremental pull requests to improve documentation or waiting till there is functionality to pull in? I was presuming small easy to understand changes were better - but now i need to know how to resolve the conflict when my  forked master has moved along significantly since then.

**Comment by rob-metalinkage** [comment]
I've synced up the fork with the head of the master branch and resolved the conflict (in requirements.txt)  so can we just discard this pull request and start again with the latest updates?

Still need to work out how to do a pull request for a subset of changes :-(

**Pull request closed without merging**